src/components/ui/ControlBar/ControlBarItem.tsx

```diff
@@ -22,8 +22,12 @@ export interface ControlBarButtonProps
   label: string;
   /** The items to render in a popover menu. When passed, the button will render an arrow to open or close a popover menu. Refer to [PopOverItem](/?path=/docs/ui-components-popover--basic-pop-over-menu) */
   popOver?: PopOverItemProps[] | null;
+  /** Defines the placement of PopOver menu. */
+  popOverPlacement?: Placement;
   /**  Apply this prop to receive visual feedback that the button is 'active' */
   isSelected?: boolean;
+  /** Use children to define an alternative to popOver prop with a custom set of elements to be rendered into the popover */
+  children?: ReactNode | ReactNode[];
```

Comment:
Should we just have this interface extend `React.FC` instead of explicitly defining children?

Or was this so we could explicitly call it out in the documentation?

Reply:
It was the latter. I could add something to that effect to the documentation, but my concern is that it'll make it more confusing.

Reply:
Yeah that makes sense, that sounds good to me